frontend/packages/react-app/package.json

```diff
@@ -65,6 +65,7 @@
     "ethers": "^5.0.17",
     "graphql": "^14.6.0",
     "ipfs-http-client": "^48.1.3",
+    "moment": "^2.29.1",
```

Comment:
momentjs is treated as legacy by themselves. 
https://momentjs.com/docs/#/-project-status/

You can use other recommendations.
https://momentjs.com/docs/#/-project-status/recommendations/

Reply:
Fixed